integrated_channels/blackboard/exporters/learner_data.py

```diff
@@ -34,8 +34,9 @@ def get_learner_data_records(
         enterprise_customer_user = enterprise_enrollment.enterprise_customer_user
         if enterprise_customer_user.user_email is None:
             LOGGER.debug(
-                'No learner data was sent for user [%s] because a Blackboard user ID could not be found.',
-                enterprise_customer_user.username
+                'No learner data was sent for LMS User [%s] because Blackboard User ID not found for customer [%s]',
+                enterprise_customer_user.user_id,
```

Comment:
So I appreciate and understand removing the user email since that is PII, however, since we now have an additional step to be able to tell what user is affected or find the user affected, I'd prefer a value that we can easily hunt down. Ie I'd prefer if we could use the UUID here.

If I search by UUID, I will find all relevant logs showing this user's journey. However, if I use their ID I will get *a lot* of irrelevant logs that don't even have anything to do with a user.

Reply:
I can add the Enteprise Customer UUID here.